src/views/CoCurricularTranscript/index.js

```diff
@@ -348,6 +354,15 @@ export default class Transcript extends Component {
           </Card>
         </div>
       );
+    } else if (this.state.profile.PersonType?.includes('fac')) {
+      return (
+        <GordonLimitedAvailability
+          pageName="Experience Transcript"
+          backToLocation="My Profile"
+          backToLink="/myprofile"
+          availableTo="students and alumni"
+        />
+      );
```

Comment:
I would love to see this cleaned up so we only check `PersonType` once, checking this above, before the main content.

Reply:
done.